### PR TITLE
The date of birth of the child has been set in the format 'dd.MM.YYYY…

### PR DESCRIPTION
**Pull request by gmakeev**
…' during adding children to workshop.

### CODE REVIEW

File: src/app/shared/components/child-card/child-card.component.ts [hunk 1]
```diff
@@ -11,17 +12,20 @@ export class ChildCardComponent implements OnInit {
 
   public below = 'below';
   public childFullName: string;
+  public childAge: string;
 
   @Input() child: Child;
+  //@Input() application: Application;
```
Comment (litvinets) [inline src/app/shared/components/child-card/child-card.component.ts:18]: why?

File: src/app/shared/components/child-card/child-card.component.html [hunk 2]
```diff
@@ -14,20 +14,19 @@
...
-            matTooltipClass="tooltip" 
+            matTooltipClass="tooltip"
             [matTooltipPosition]="below"
           >{{child.lastName}} {{child.firstName}} {{child.middleName}}</mat-card-title>
         </mat-card-header>
         <mat-card-content fxLayout="column" fxLayoutAlign="space-between start">
           <p class="card_text">
-            <mat-icon class="card_icon">calendar_today</mat-icon> {{child.dateOfBirth | date:
-            'shortDate'}}
+            <mat-icon class="card_icon">calendar_today</mat-icon> {{child.dateOfBirth | date: 'dd.MM.YYYY'}} ({{ childAge }})
```
Comment (litvinets) [inline src/app/shared/components/child-card/child-card.component.html:26]: date: constants.SHORT_DATE_FORMAT

File: src/app/shared/components/person-card/person-card.component.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
   </div>
   <p>{{ card.lastName }} {{ card.firstName }} {{ card.middleName }} </p>
   <label class="card-label">Дата народження</label>
-  <p> {{ card.dateOfBirth }} </p>
+  <p>{{card.dateOfBirth | date: 'dd.MM.YYYY'}}</p>
```
Comment (litvinets) [inline src/app/shared/components/person-card/person-card.component.html:19]: date: constants.SHORT_DATE_FORMAT

File: src/app/shared/utils/utils.ts [hunk 4]
```diff
@@ -33,7 +33,11 @@ export class Util {
   public static getDeclensionYear(year: number) : string {
     let ageString;
     let lastDigit = year % 10;
-    (lastDigit === 1) ? ageString = 'рік' : (lastDigit > 1 && lastDigit < 5) ? ageString = 'роки' : ageString = 'років';
+    if (year === 11) {
+      ageString = 'років';
+    } else {
+      (lastDigit === 1) ? ageString = 'рік' : (lastDigit > 1 && lastDigit < 5) ? ageString = 'роки' : ageString = 'років';
```
Comment (litvinets) [inline src/app/shared/utils/utils.ts:39]: (lastDigit === 1 && year !== 11)

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**